sdk/notificationhubs/notification-hubs/vitest.browser.unit.config.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig, mergeConfig } from "vitest/config";
+import viteConfig from "../../../vitest.browser.shared.config.ts";
+
+export default mergeConfig(
```

Comment:
nit: would it be better to pass an customization option to our shared `viteConfig` and do the merge there? reduce a little code duplication.